lib/class-wp-rest-image-editor-controller.php

```diff
@@ -36,35 +42,88 @@ public function __construct() {
 	public function register_routes() {
 		register_rest_route(
 			$this->namespace,
-			$this->rest_base . '/apply',
+			$this->rest_base,
 			array(
 				array(
 					'methods'             => WP_REST_Server::EDITABLE,
 					'callback'            => array( $this, 'apply_edits' ),
 					'permission_callback' => array( $this, 'permission_callback' ),
 					'args'                => array(
-						'x'        => array(
-							'type'     => 'number',
-							'minimum'  => 0,
-							'required' => true,
-						),
-						'y'        => array(
-							'type'     => 'number',
-							'minimum'  => 0,
-							'required' => true,
-						),
-						'width'    => array(
-							'type'     => 'number',
-							'minimum'  => 1,
-							'required' => true,
-						),
-						'height'   => array(
-							'type'     => 'number',
-							'minimum'  => 1,
-							'required' => true,
-						),
-						'rotation' => array(
-							'type' => 'integer',
+						array(
```

Comment:
This is a pretty large array and makes this extremely hard to read. This should be broken into it's own methon on the class similar to `get_collection_params`